### ISSUE TITLE
Write a short Wiki page explaining dispaly.

### ISSUE THREAD

**Issue by dpc** [vadixidav/evomata11#1]
I glanced through the code, I like the screenshots, I like the concept, but when I'm running it I don't really understand what is going on. :)

**Comment by vadixidav** [comment]
Yeah, everyone always asks me what it means, and that might be a good way to explain it.
